src/pages/event-schema.ts

```diff
@@ -14,8 +14,20 @@ export const eventDateSchema = Joi.date()
     "date.max": `Date must be before ${maxDate.toISOString().split("T")[0]}`,
   });
 
+export const endDateSchema = Joi.date()
+  .iso()
+  .allow("")
+  .min(Joi.ref("startDate"))
+  .max(maxDate)
+  .messages({
+    "date.format": `Date must be a real date`,
+    "date.min": `End date must not be before start date`,
+    "date.max": `Date must be before ${maxDate.toISOString().split("T")[0]}`,
+  });
+
 export const eventSchema = Joi.object({
```

Comment:
feels like this schema is now describing a stage rather than an event as it has a startDate and an endDate (where events have only eventDate) so might be better to call it stageSchema